src/values/TimeValue.js

```diff
@@ -35,26 +105,107 @@ var SELF = dv.TimeValue = util.inherit( 'DvTimeValue', PARENT, function( value )
 	 * @return {string}
 	 */
 	getSortKey: function() {
-		return this.getValue().iso8601();
+		return this._getISO8601();
 	},
 
 	/**
 	 * @inheritdoc
 	 *
-	 * @return {time.Time}
+	 * @return {Object}
 	 */
 	getValue: function() {
-		return this._value;
+		return this.toJSON();
+	},
+
+	/**
+	 * @since 0.7
+	 *
+	 * @return {number}
+	 */
+	getYear: function() {
+		return this._time.year;
+	},
+
+	/**
+	 * @since 0.7
+	 *
+	 * @return {number}
+	 */
+	getMonth: function() {
+		return this._time.month;
+	},
+
+	/**
+	 * @since 0.7
+	 *
+	 * @return {number}
+	 */
+	getDay: function() {
+		return this._time.day;
+	},
+
+	/**
+	 * @since 0.7
+	 *
+	 * @return {number}
+	 */
+	getHour: function() {
+		return this._time.hour;
+	},
+
+	/**
+	 * @since 0.7
+	 *
+	 * @return {number}
+	 */
+	getMinute: function() {
+		return this._time.minute;
+	},
+
+	/**
+	 * @since 0.7
+	 *
+	 * @return {number}
+	 */
+	getSecond: function() {
+		return this._time.second;
 	},
 
 	/**
 	 * @inheritdoc
 	 */
 	equals: function( value ) {
-		if ( !( value instanceof SELF ) ) {
-			return false;
+		if( value instanceof SELF ) {
+			return true;
```

Comment:
Huh? This is enough to decide if two objects are equal? I think this is not correct but not covered by a test, unfortunately.